docs/event-bus/08-01-configure-default-channelprovisioner.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Subscription
+type: Configure default ClusterChannelProvisioner
+---
+
+Kyma comes with NATSS as its default ClusterChannelProvisioner(See [default-channel-webhook](../../resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml) ConfigMap). Other than NATSS, one can use any default channel provisioner of one's choice. In order to add a in-memory-channel provisioner follow this [guide](https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel).
```

Comment:
Shall we create sections for each channel provisioner so that they can be extended?

Reply:
Created sections now